src/Client/Adapter/Curl.php

```diff
@@ -209,6 +209,13 @@ public function connect($host, $port = 80, $secure = false)
             }
         }
 
+        if (isset($this->config['sslcafile']) && $this->config['sslcafile']) {
```

Comment:
@mlocati this is not the point, I don't want to change the behaviour of any settings. I would like to prevent wrong settings. Anyway, reading the [CURLOPT_SSL_VERIFYPEER](https://curl.haxx.se/libcurl/c/CURLOPT_SSL_VERIFYPEER.html) specification it seems that it works already as is. If it's false, the CA settings are not used anyway.
I think we can just leave the settings as you proposed without any extra check and exceptions. We should only remember to document this behaviour in the documentation.